packages/ui-progress/src/ProgressCircle/index.tsx

```diff
@@ -135,20 +132,17 @@ class ProgressCircle extends Component<
 
     const valueText =
       typeof formatScreenReaderValue === 'function' &&
-      formatScreenReaderValue({ valueNow, valueMax })
+      formatScreenReaderValue({ valueNow: valueNow!, valueMax: valueMax! })
```

Comment:
I think this needs a bugfix here. Unlike ProgressBar, the original type of this prop allows a func OR node, but the node is never processed. So I think we should make it the same type as ProgressBar and only accept a function that returns a string.